L1Trigger/L1TGlobal/src/EnergySumZdcTemplate.cc

```diff
@@ -0,0 +1,104 @@
+/**
+ * \class EnergySumZdcTemplate
+ *
+ *
+ * Description: L1 Global Trigger energy-sum template.
+ *
+ * Implementation:
+ *    <TODO: enter implementation details>
+ *
+ * \author: Vasile Mihai Ghete - HEPHY Vienna
```

Comment:
Maybe update the description and author (or simply remove it) ?

Reply:
I've updated this text to now attribute to myself an @elfontan but also to credit the original author + point to original EnergySumTemplate.cc as this is largely a clone of existing work. In latest commit